src/main/java/no/unit/nva/model/Contributor.java

```diff
@@ -1,27 +1,43 @@
 package no.unit.nva.model;
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.exceptions.MalformedContributorException;
 
 import java.util.List;
 import java.util.Objects;
 
+import static java.util.Objects.isNull;
+
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Contributor {
 
+    public static final String CORRESPONDING_AUTHOR_EMAIL_MISSING =
+            "The Contributor is corresponding author, but no email for correspondence is set";
     private Identity identity;
     private List<Organization> affiliations;
     private Role role;
     private Integer sequence;
+    private boolean correspondingAuthor;
+    private String email;
 
     public Contributor() {
 
     }
 
-    private Contributor(Builder builder) {
+    private Contributor(Builder builder) throws MalformedContributorException {
+        if (isCorrespondAuthorWithEmail(builder)) {
```

Comment:
I have a suggestion, make a Constructor annotated with JsonCreator that has all obligatory fields and move the validation there. We want the validation to be active for normal deserialization too, not just for the Builder's Constructor.